skew/arn/__init__.py

```diff
@@ -16,7 +16,7 @@
 import logging
 import re
 
-from six.moves import zip_longest
+from botocore.vendored.six.moves import zip_longest
```

Comment:
I have generally felt that relying on vendored transitive dependencies is a bad thing, since in theory botocore could drop the six dependency.